front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -46,34 +60,17 @@ const ScenarioDescription = ({
         >
           <Pencil />
         </button>
-        <button
-          type="button"
-          className="scenario-details-modify-button"
-          aria-label={t('toggleTimetable')}
-          onClick={() => collapseTimetable()}
-        >
-          <ChevronLeft />
-        </button>
       </div>
-      <div className="row">
-        <div className="col-md-6">
-          <div className="scenario-details-infra-name">
-            <img src={infraLogo} alt="Infra logo" className="infra-logo mr-2" />
-            {infra && <InfraLoadingState infra={infra} />}
-            <span className="scenario-infra-name">{scenario.infra_name}</span>
-            <small className="ml-auto text-muted">ID {scenario.infra_id}</small>
-          </div>
-        </div>
-        <div className="col-md-6">
-          <div className="scenario-details-electrical-profile-set">
-            <span className="mr-2">
-              <GiElectric />
-            </span>
-            {scenario.electrical_profile_set_id
-              ? scenario.electrical_profile_set_id
-              : t('noElectricalProfileSet')}
-          </div>
-        </div>
+      <div className="scenario-details-electrical-profile-set">
+        {scenario.electrical_profile_set_id
+          ? scenario.electrical_profile_set_id
+          : t('noElectricalProfileSet')}
+      </div>
+
+      <div className="scenario-details-infra-name">
+        {t('infrastructure')} :&nbsp;{infra && <InfraLoadingState infra={infra} />}
```

Comment:
Why nbsp after the colon but not before ?

Reply:
If I don't put nbsp after the colon, there's no space, whereas before, they're put natively.